pbm/storage/s3/s3.go

```diff
@@ -143,6 +144,10 @@ func (s *S3) Save(name string, data io.Reader, sizeb int) error {
 			}
 		}
 
+		maxUploadParts := s3manager.MaxUploadParts
+		if s.opts.MaxUploadParts > 0 {
+			maxUploadParts = s.opts.MaxUploadParts
+		}
```

Comment:
The same as with [the previous PR](https://github.com/percona/percona-backup-mongodb/pull/615) I'd move it to the Cast()

Cheers!

Reply:
I've move the code inside `Cast()`. Is that what you wanted?